File: backend-java/src/main/java/associados/api/dto/AssociadoForm.java
package associados.api.dto;

import associados.api.model.Categoria;

import java.time.LocalDate;

public record AssociadoForm(
        Long id,
        String nome,
        String observacao,
        Categoria categoria,
        LocalDate dataNascimento,
        String naturalidade,
        String estadoNatural,
        String estadoCivil,
        String nacionalidade,
        String endereco,
        String bairro,
        String cidade,
        String estado,
        String telefone,
        int mesPago,
        int anoPago
) {
}

File: backend/src/main/kotlin/associados/api/dto/AssociadoForm.kt
package associados.api.dto

import associados.api.model.Categoria
import java.time.LocalDate

data class AssociadoForm(
    val nome: String,
    val observacao: String,
    val categoria: Categoria,
    val dataNascimento: LocalDate,
    val naturalidade: String,
    val estadoNaturalidade: String,
    val nacionalidade: String,
    val estadoCivil: String,
    val endereco: String,
    val bairro: String,
    val cidade: String,
    val estado: String,
    val telefone: String,
    val mesPago: Int,
    val anoPago: Int
)